core/test/vault-test.ts

```diff
@@ -0,0 +1,390 @@
+            collateralType: 'ETH-C',
+            address: '0x50A10041A113f8dF54EAf7f9f49F03c1Eb6B45bc',
+            network: TEST_NETWORK,
+
+            lastSyncedAt: new Date(),
+            nextPriceChange: new Date('2022-06-13T11:00:00.000Z'),
+
+            liquidationRatio: new BigNumber(1.7),
+            collateralizationRatio: new BigNumber('1.698313241866926788910221381'),
+            proximityToLiquidation: new BigNumber('-0.000993196126304112564617121'),
+            liquidationPenaltyRatio: new BigNumber('1.13'),
+            minimalAuctionedDai: new BigNumber('5000'),
+            id: 27435,
+
+            incentiveCombinedDai: new BigNumber('308.851309396336731694851'),
+            incentiveRelativeDai: new BigNumber('8.851309396336731694851'),
+            incentiveConstantDai: new BigNumber('300'),
+            grossProfitDai: new BigNumber('308.851309396336731694851'),
+            netProfitDai: new BigNumber('307.843172251262360274006580599088120101895341629034'),
+            stabilityFeeRate: new BigNumber('1.013829372344109723319869134'),
+            minUnitPrice: new BigNumber('721.804640220588235294117647058'),
+            maximumProtocolDebtDai: new BigNumber('100000000'),
+            currentProtocolDebtDai: new BigNumber(0),
+            currentCollateralDebtDai: new BigNumber('105882.900090330108539882708255451806124201305822084'),
+            maximumCollateralDebtDai: new BigNumber(35000000),
+            transactionFeeLiquidationEth: new BigNumber('0.000669987000446658'),
+            transactionFeeLiquidationDai: new BigNumber('1.130545198449229148108694994096671174'),
+            debtDai: new BigNumber('8973.717449711589422115275593184791275895341629034'),
+            collateralAmount: new BigNumber('12.42'),
+            currentUnitPrice: new BigNumber('1227.067888375'),
+            nextUnitPrice: new BigNumber('1208.0159951'),
+            initialDebtDai: new BigNumber('8851.309396336731694851'),
+        };
+        compareVaultTransactionsNotLiquidated(expectedObject, vaultTransactionNotLiquidated);
+        await liquidateVault(
+            TEST_NETWORK,
+            vaultTransactionNotLiquidated.collateralType,
+            vaultTransactionNotLiquidated.address,
+            HARDHAT_PUBLIC_KEY
+        );
+        const updatedVault = (await fetchVault(TEST_NETWORK, 27435)) as Vault;
+        expect(updatedVault.collateralAmount.toFixed()).to.eq('0');
+        expect(updatedVault.initialDebtDai.toFixed()).to.eq('0');
+        const updatedVaultTransaction = (await getVaultTransaction(
+            TEST_NETWORK,
+            updatedVault
+        )) as VaultTransactionLiquidated;
+        expect(updatedVaultTransaction.state).to.eq('liquidated');
+        const liquidationAuction = await fetchAuctionByCollateralTypeAndAuctionIndex(TEST_NETWORK, 'ETH-C', 191);
+        expect(liquidationAuction.collateralAmount).to.deep.equalInAnyOrder(expectedObject.collateralAmount);
```

Comment:
Please compare `liquidationAuction` with the value from `vaultTransactionNotLiquidated` for both:
- [ ] `collateralAmount`
- [ ] `debtDai`

Reply:
Good point, opened up the thing where the gross profit of liquidation is not taking the liquidation penalty int account

Reply:
The first part was not addressed
```suggestion
        expect(liquidationAuction.collateralAmount).to.deep.equalInAnyOrder(vaultTransactionNotLiquidated.collateralAmount);
```